src/configs/env.ts

```diff
@@ -457,6 +502,22 @@ export const SUPERSET_IRS_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE =
   process.env.REACT_APP_SUPERSET_IRS_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE || '0';
 export type SUPERSET_IRS_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE = typeof SUPERSET_IRS_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE;
 
+export const SUPERSET_IRS_LITE_DISTRICT_PERFORMANCE_REPORT_SLICE =
+  process.env.REACT_APP_SUPERSET_IRS_LITE_DISTRICT_PERFORMANCE_REPORT_SLICE || '0';
+export type SUPERSET_IRS_LITE_DISTRICT_PERFORMANCE_REPORT_SLICE = typeof SUPERSET_IRS_LITE_DISTRICT_PERFORMANCE_REPORT_SLICE;
+
+export const SUPERSET_IRS_LITE_DATA_COLLECTORS_PERFORMANCE_REPORT_SLICE =
+  process.env.REACT_APP_SUPERSET_IRS_LITE_DATA_COLLECTORS_PERFORMANCE_REPORT_SLICE || '0';
+export type SUPERSET_IRS_LITE_DATA_COLLECTORS_PERFORMANCE_REPORT_SLICE = typeof SUPERSET_IRS_LITE_DATA_COLLECTORS_PERFORMANCE_REPORT_SLICE;
+
+export const SUPERSET_IRS_LITE_SOP_PERFORMANCE_REPORT_SLICE =
+  process.env.REACT_APP_SUPERSET_IRS_LITE_SOP_PERFORMANCE_REPORT_SLICE || '0';
+export type SUPERSET_IRS_LITE_SOP_PERFORMANCE_REPORT_SLICE = typeof SUPERSET_IRS_LITE_SOP_PERFORMANCE_REPORT_SLICE;
+
+export const SUPERSET_IRS_LITE_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE =
+  process.env.REACT_APP_SUPERSET_IRS_LITE_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE || '0';
+export type SUPERSET_IRS_LITE_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE = typeof SUPERSET_IRS_LITE_SOP_BY_DATE_PERFORMANCE_REPORT_SLICE;
+
```

Comment:
Do we have IRS-Lite performance reports?  If not then we should get rid of these.

Reply:
Done